content/blog/2023-02-16-nextjs-tutorial.md

```diff
@@ -64,7 +64,7 @@ In order to create a new Next.js project using the Flowershow template, you can
 Run the following command in the terminal and then pass a name of your project when prompted (we'll name it `my-flowershow-app` for the purpose of this tutorial).
 
 ```shell
-npx create-next-app@latest --example https://github.com/flowershow/flowershow/tree/main/packages/template
+npx create-next-app@latest --example hhttps://github.com/datopian/flowershow/tree/main
```

Comment:
@Gutts-n there is a typo in the protocol of the uri 😄  have you tested running this command? Do you think we need to specify branch or it can be simply https://github.com/datopian/flowershow ?